rd_workshop/styles/blocks/header.scss

```diff
@@ -0,0 +1,132 @@
+.header {
+    box-sizing: border-box;
+    background-image: url(../img/home.png);
+    width: 1903px;
```

Comment:
why not to use 100%?
test your website for smaller screens, the block it's outside